x: 0.5
y: 0.5
width: 4
height: 4.5
trans_vel: 1.2
rot_vel: 1.4
frequency: 10
